worker/apiserver/manifold.go

```diff
@@ -216,6 +222,11 @@ func (config ManifoldConfig) start(context dependency.Context) (worker.Worker, e
 		return nil, errors.Trace(err)
 	}
 
+	var jwtParserGetter jwtparser.Getter
```

Comment:
See comments on parser about this.

Reply:
We ideally want a type that we can pull from a package that isn't the other worker to decouple the two.